### PR TITLE
[GSK-1419][GSK-1383] Make dataset, model as suite input in `Suite` and upload them automatically

### PR DESCRIPTION
**Pull request by Inokinoki**
## Related Issue

[GSK-1383] Make dataset and model as suite input, when generating the test suite from scan
[GSK-1419] Add the possibility to set default `run` parameters to a test suite

## Type of Change

- [ ] 📚 Examples / docs / tutorials / dependencies update
- [ ] 🔧 Bug fix (non-breaking change which fixes an issue)
- [x] 🥂 Improvement (non-breaking change which improves an existing feature)
- [ ] 🚀 New feature (non-breaking change which adds functionality)
- [ ] 💥 Breaking change (fix or feature that would cause existing functionality to change)
- [ ] 🔐 Security fix

## Checklist

- [ ] I've read the [`CODE_OF_CONDUCT.md`](https://github.com/Giskard-AI/ai-inspector/blob/master/CODE_OF_CONDUCT.md) document.
- [ ] I've read the [`CONTRIBUTING.md`](https://github.com/Giskard-AI/ai-inspector/blob/master/CONTRIBUTING.md) guide.
- [ ] I've updated the code style using `make codestyle`.
- [ ] I've written tests for all new methods and classes that I created.
- [ ] I've written the docstring in Google format for all the methods and classes that I used.

### CODE REVIEW

**Comment by mattbit** [comment]
@Inokinoki is this ready to review?

File: giskard/core/suite.py [hunk 1]
```diff
@@ -311,22 +311,28 @@ class Suite:
...
+            A dictionary containing the default parameters for the tests in the suite.
     """
 
     id: int
     tests: List[TestPartial]
     name: str
+    default_params: Dict[str, Any]
 
-    def __init__(self, name=None) -> None:
+    def __init__(self, name=None, **default_params) -> None:
```
Comment (mattbit) [inline giskard/core/suite.py:323]: Can we use a normal argument instead of the catch-all `**kwargs`?

**Review by mattbit — COMMENTED** [review]
Just a minor comment

File: giskard/core/suite.py (hunk 1 above)
Comment (Inokinoki) [reply giskard/core/suite.py]: Sure, it's done.

I realize that the parameters like `name=""` will be ignored otherwise.

**Review by mattbit — APPROVED** [review]

**Pull request merged**